google/sheets/readSpreadsheet.py

```diff
@@ -38,29 +39,29 @@ async def main():
                 cell_reference = get_cell_reference(row_idx, col_idx)
                 cell_values[cell_reference] = value
 
-        if len(cell_values) > 10:
-            try:
-                gptscript_client = GPTScript()
-                dataset = await gptscript_client.create_dataset(
-                    os.getenv("GPTSCRIPT_WORKSPACE_DIR"),
-                    f"{spreadsheet.id}_data",
-                    f"data for Google Sheet with ID {spreadsheet.id}",
-                )
-
-                for cell_reference, value in cell_values.items():
-                    await gptscript_client.add_dataset_element(
-                        os.getenv("GPTSCRIPT_WORKSPACE_DIR"), dataset.id, cell_reference, value if value != "" else " "
-                    )
-                print(f"Dataset created with ID {dataset.id} with {len(cell_values)} elements")
-                return
-            except Exception:
-                pass  # Ignore errors if we got any, and just print the results.
-
-        for cell_reference, value in cell_values.items():
-            if show_cell_ref:
-                print(f"{cell_reference}: {value}")
-            else:
-                print(value)
```

Comment:
Note to self: remove the corresponding tool param for this if it is no longer needed